cmd/policy.go

```diff
@@ -103,7 +103,7 @@ var (
   # See https://hub.cloudquery.io for additional policies.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			source := args[0]
-			c, err := console.CreateClient(cmd.Context(), getConfigFile(), true, nil, instanceId)
```

Comment:
what is the use for `allowDefaultConfig` argument? should we remove it completely (assume `false`)?